### PR TITLE
Update markets for Berlin (published 09.04.2018).

### PR DESCRIPTION
**Pull request by johnjohndoe**

### CODE REVIEW

File: cities/berlin.json [hunk 1]
```diff
@@ -2288,6 +2320,22 @@
...
+        {
+            "geometry": {
+                "coordinates": [
+                    13.308666,
+                    52.4669384
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "<164 Markt (Verkauf von Baby- und Kindersachen)",
```
Comment (dirkschumacher) [inline cities/berlin.json]: Why is there a special character "<"? Or is that the real name?

File: preprocessing/README.md [hunk 2]
```diff
@@ -8,7 +8,9 @@ convert it into GeoJSON format.
 
 1. Download the latest raw market data and put in into the *raw*
 folder as *markets-berlin.json*.
-2. Convert the raw market data by running the following command from
+2. Manually reformat the downloaded JSON file using the [*Pretty JSON* plugin][pretty-json]
```
Comment (dirkschumacher) [inline preprocessing/README.md]: Why do you first prettyfy it and then convert it?

File: cities/berlin.json (hunk 1 above)
Comment (johnjohndoe) [reply cities/berlin.json]: Looking at the source file in [line 4224](https://github.com/wo-ist-markt/wo-ist-markt.github.io/pull/262/files#diff-077b45c932e5588fa665a45630872a0eR4224) it seems to be a typo. I wrote a message to the data provider to clarify/fix it.

File: preprocessing/README.md (hunk 2 above)
Comment (johnjohndoe) [reply preprocessing/README.md]: Good point. I explained my reasoning in the updated commit.

File: cities/berlin.json (hunk 1 above)
Comment (johnjohndoe) [reply cities/berlin.json]: I received an answer:
> Es soll tatsächlich das "kleiner-Zeichen" darstellen. Wegen Babysachen. Also kleiner als Größe 164.

**Pull request merged**